### PR TITLE
 #25 Implementer les endpoints de l'objet Log

### PR DESCRIPTION
**Pull request by MentalistHB**

### CODE REVIEW

File: backend/src/main/java/com/ambacam/model/Log.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+@Entity
+public class Log implements Serializable {
+
+	private static final long serialVersionUID = -8019398584857375323L;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private Long id;
+
+	@Column(columnDefinition = "varchar", nullable = false)
```
Comment (Chombon) [inline backend/src/main/java/com/ambacam/model/Log.java]: In the swagger this property can be null

File: backend/src/main/java/com/ambacam/rest/ApiConstants.java [hunk 2]
```diff
@@ -9,4 +9,7 @@
 	public static final String STATUS_REQUETE_COLLECTION = "/status-requetes";
 
 	public static final String STATUS_REQUETE_ITEM = STATUS_REQUETE_COLLECTION + "/{statusRequeteId}";
+
+	public static final String LOG_COLLECTION = "/logs";
```
Comment (Chombon) [inline backend/src/main/java/com/ambacam/rest/ApiConstants.java]: In the swagger the path is: "/log/search"

**Comment by Chombon** [comment]
Also squash and rebase please.

File: backend/src/main/java/com/ambacam/rest/ApiConstants.java (hunk 2 above)
Comment (MentalistHB) [reply backend/src/main/java/com/ambacam/rest/ApiConstants.java]: You are right,

but /log/search is an error in the swagger. It should be /logs/search

**Pull request merged**